packages/dashboard-core-plugins/src/MarkdownPlugin.tsx

```diff
@@ -52,16 +32,12 @@ export function MarkdownPlugin(props: MarkdownPluginProps): JSX.Element | null {
       const openedMarkdowns = panelManager.getOpenedPanelConfigsOfType(
         MarkdownPanel.COMPONENT
       );
-      const closedMarkdowns = panelManager.getClosedPanelConfigsOfType(
-        MarkdownPanel.COMPONENT
-      );
       const usedTitles = openedMarkdowns.map(markdown => markdown.title ?? '');
       const panelTitle =
         title != null && title !== ''
           ? title
           : MarkdownUtils.getNewMarkdownTitle(usedTitles);
-      const content =
-        closedMarkdowns.length > 0 ? null : MarkdownUtils.DEFAULT_CONTENT;
+      const content = MarkdownUtils.DEFAULT_CONTENT;
```

Comment:
We don't want to do this - right now this change is causing a regression, which is re-opening closed markdown widgets.

If you open a markdown panel, edit it to add some text, then close it, and then open a new markdown panel, you get a starting page prompting you to re-open one of the previously closed markdown panels or to just create a new one:
<img width="497" alt="image" src="https://github.com/deephaven/web-client-ui/assets/4505624/e12d369a-8d13-4181-8c49-9772dbee837d">

I think what you _should_ do is just set `null` for content from here, then in MarkdownPanel is where it will actually decide whether to show the starting page or the default content (when it calls `getClosedMarkdowns`).

Reply:
Just pushed changes, if I understood correctly when you said 

> then in MarkdownPanel is where it will actually decide whether to show the starting page or the default content (when it calls getClosedMarkdowns).

It now works such that on initial opening of a Markdown, you get this panel, given that when `getClosedMarkdowns` is called in MarkdownPanel, the condition for the starting page appearing is whether the state isStatePageShown is set to true:
![image](https://github.com/deephaven/web-client-ui/assets/69530774/49a6bf29-0800-4daa-81c2-0a35ea9e14cc)


However, after refreshing after editing, we see the default content: 

![image](https://github.com/deephaven/web-client-ui/assets/69530774/ad73e2ce-7f72-4ded-ac1e-d5677d195fd8)

Reply:
*** Something seems to be breaking, need to inspect, will update ***